packages/amplication-client/src/Plugins/PluginsCatalogItem.tsx

```diff
@@ -137,7 +137,7 @@ function PluginsCatalogItem({
       <div className={`${CLASS_NAME}__row `}>
         <span className="spacer" />
         <span className={`${CLASS_NAME}__repo`}>
-          <a href={plugin.github} target="github_plugin">
+          <a href={plugin?.github} target="github_plugin">
```

Comment:
@Meyazhagan Why did you add the ? mark?

Reply:
Plugin is `undefined` for some cases (like while loading or no plugin definition) so I handled for those cases